base/BUILD

```diff
@@ -36,3 +36,28 @@ structure_test(
     config = "testdata/base.yaml",
     image = ":base",
 )
+
+# For convenience, rename busybox-x86_64 to busybox.
+genrule(
+    name = "_busybox",
+    srcs = ["@busybox//file"],
+    outs = ["busybox"],
+    cmd = "cp $(<) $(@)",
+    visibility = ["//visibility:private"],
+)
+
+# A debug image with busybox available.
+docker_build(
```

Comment:
> Do you want to start adding ":debug" targets to each of the base runtimes that includes this and publishing them?

I think for now we should probably just document how to drop the tarball into any other runtime. It's one line and probably makes more sense than doubling our images.